new_jersey/ver2/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <title>New Jersey</title>
+</head>
+<body>
+<section class="container">
```

Comment:
The semantic tag 'section' represents a standalone section in the HTML document. Generally, containers and wrappers are created using the <div> tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
Would the variant below be correct? I mean if element such as container or wrapper are created using `<div>`, but the main element - widget - would be in `<section>`. Just I thought that widget is the standalone section, that's why I did so. Or using any semantic tag like `<header>`, `<section>`, etc inside `<div>` isn't correct?

```
<body> 
    <div class="container">
        <section class="widget">
```

Reply:
Yep, the variant you suggested is correct. In HTML, there are no explicit rules on which tags can be put inside which, but there is one recommendation to not put the same action tags inside one another (`<button>, <input>, <form>`, etc.):
```
<button>
    <button></button>
</button>
```
The code snippet above will render 2 buttons next to each other instead of putting the second one inside of the first one.

These 'general' HTML recommendations were upgraded to strict rules in XHTML: https://it.wikireading.ru/662